### ISSUE TITLE
Can IOS do real inference ?

### ISSUE THREAD

**Issue by GwynbleiddzRivia** [mit-han-lab/inter-operator-scheduler#11]
Hi, i have a question here.

Can IOS do real inference ? 
For example, I create an IOS graph and it has random weights. Then, by using a random input, can IOS correctly go through all the operator execution?

**Comment by yaoyaoding** [comment]
Hi,

I do not recommend using the code in this repo for actual inference, but suggesting you give framework [raf](https://github.com/awslabs/raf) a try :) 
I added the [multi-stream support](https://github.com/awslabs/raf/blob/main/docs/wiki/3_dev_guide/pass/Multi-CUDA-Stream-Schedule.md) and some inter-operator schedules (including IOS) to it during my internship at Amazon.

The code in this repo is used for demonstrating ideas, but not for actual deployment. By design, the code in this repo should be able to produce the correct results (I had checked those during the development), but I can not spare much time on maintaining it (e.g., fixing bugs).

Best,
Yaoyao
